### ISSUE TITLE
Adding an about us page to the website. [GSSoC'23]

### ISSUE THREAD

**Issue by 1Shubham7** [J0SAL/Decentralized-Expense-Tracker#101]
### Is your feature request related to a problem? Please describe.

The website doesn't have a about us page, I can create an about us page for the website, which will make the website look better and more user friendly.

### Describe the solution you'd like

I will create an about us page.

### Additional context

Please assign me this issue under GSSoC'23.

**Comment by 1Shubham7** [comment]
Please assign me this issue under GSSoC'23

**Comment by J0SAL** [comment]
@1Shubham7  All the info about the application is conveyed via the footer and title, what additional things can we add via the About us page?

**Comment by avoxaya** [comment]
About us page can further explain the cause and motive of the web page and also It shows a more direct and professional approach from the business point of view. Please assign me this issue under GSSoC'23

**Comment by 1Shubham7** [comment]
yup, since you explained the reason so well, @J0SAL , please assign me this issue.

**Comment by J0SAL** [comment]
As @1Shubham7 was the one to create the issue, I'm assigning it to hi; but I liked @avoxaya's response, I think @avoxaya can help in reviewing the PR, once it is ready
